Dockerfile

```diff
@@ -53,3 +53,7 @@ ENV PATH="${DPCPP_HOME}/llvm/build/bin:${PATH}"
 ENV LD_LIBRARY_PATH="${DPCPP_HOME}/llvm/build/lib:${LD_LIBRARY_PATH}"
 
 RUN apt-get install -y ocl-icd-* opencl-headers intel-opencl-icd
+
+RUN wget https://github.com/oneapi-src/oneDPL/archive/refs/tags/oneDPL-2021.7.1-release.tar.gz
```

Comment:
Changing it to `https://github.com/oneapi-src/oneDPL/archive/refs/tags/oneDPL-2022.0.0-release.tar.gz` fixed build issue for me